examples/Kittens/Sample/AppDelegate.m

```diff
@@ -24,3 +24,15 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
 }
 
 @end
+
+@implementation ASConfiguration (UserProvided)
+
++ (ASConfiguration *)textureConfiguration
+{
+  ASConfiguration *configuration = [ASConfiguration new];
+//  configuration.experimentalFeatures = ASExperimentalTextNode; 
```

Comment:
Possibly worth landing it with this enabled :). Otherwise probably delete the commented line.